### ISSUE TITLE
Verbal warning for zombies if they move when dead

### ISSUE THREAD

**Issue by adangert** [adangert/JoustMania#121]
Sometimes zombies forget they can't move, a verbal warning and a penalty might help prevent this. Such as bullets to more humans or more zombie deaths.

**Comment by martinbogo** [comment]
The whole bullet mechanic is really awkward Aaron.  Slow-moving zombies are good, but the pistol + bullet isn't as fun or effective as the molotov cocktails.

**Comment by adangert** [comment]
yeah I have an issue open for that: https://github.com/adangert/JoustMania/issues/118 needs testing but I think you are right.
